entrypoint.sh

```diff
@@ -19,12 +20,12 @@ then
 fi
 
 function cleanup() {
-  if [[ "${GITHUB_REF}" == refs/heads/* ]]
+  if [[ "${BRANCH_NAME}" == refs/heads/* ]]
   then
-    git checkout ${GITHUB_REF}
+    git checkout ${BRANCH_NAME}
   else
```

Comment:
By this change, `[[ "${BRANCH_NAME}" == refs/heads/* ]]` will be always false.
Above condition can be removed.